vhdbuilder/packer/install-dependencies.sh

```diff
@@ -818,29 +823,24 @@ configureLsmWithBpf() {
 
   # Prepend bpf to the LSM list if not already present
   if ! echo "$current_lsm" | grep -q bpf; then
-    if [ "$IS_KATA" = "true" ] || echo "$FEATURE_FLAGS" | grep -q "cvm"; then
```

Comment:
this is now supported ? what changed ?

Reply:
We had added this restriction due to test failures. I wanted to see the test failures and fix them, to remove the restriction.